distribution/lib/Standard/Database/0.0.0-dev/src/DB_Column.enso

```diff
@@ -1546,7 +1546,7 @@ type DB_Column
          unusual events like DST.
     @period Date_Time_Helpers.make_period_selector_for_column
     date_diff : (DB_Column | Date | Date_Time | Time_Of_Day) -> Date_Period | Time_Period -> DB_Column ! Invalid_Value_Type | Illegal_Argument
-    date_diff self end (period : Date_Period | Time_Period) =
+    date_diff self end (period : Date_Period | Time_Period = Time_Period.Day) =
```

Comment:
Note the peculiarity of `Time_Period.Day` (it is equal to 24h, not a calendar day - 99% of the time these are equivalent, but sometimes they are not):
![image](https://github.com/enso-org/enso/assets/1436948/1a750f77-7ae9-4268-ad46-8fcaac3d8ea1)

I'd prefer to use `Date_Period.Day` here instead.

Reply:
Changed to this.